packages/paypal-commerce-integration/.eslintrc.json

```diff
@@ -6,6 +6,7 @@
     {
       "files": ["*.ts", "*.tsx"],
       "rules": {
+        "@typescript-eslint/no-unsafe-member-access": "off",
```

Comment:
Why do you need this rule?

Reply:
https://github.com/bigcommerce/checkout-js/pull/1689/files#diff-a00ef9fab04a2b411a0abba10dc95be92b4769e4e6eb1d225f50cf3a7fa6d8b9R35


Unsafe member access .isComplete on an `any` value

Reply:
I don't think that it is a good idea to set this rule for whole package. I recommend to use 1 line disable rule for such cases or even avoid such rule and fix the issue.